src/redux/slice/greetingSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+export const fetchGreetings = createAsyncThunk(
+    "greeting/fetchGreetings",
+    async () => {
+        const response = await fetch("/api/greeting");
```

Comment:
- [ ] The client does not currently reach the server API 

<img width="878" alt="Screenshot 2023-02-04 at 12 49 22 AM" src="https://user-images.githubusercontent.com/40503704/216733216-7c2ea11a-a4d6-42d5-ad53-79bdc96fd40d.png">

Here is an example of what you should have
<img width="475" alt="Screenshot 2023-02-04 at 1 02 44 AM" src="https://user-images.githubusercontent.com/40503704/216733333-a40ec178-6ceb-4bce-ae4a-708fbfdf6090.png">